cpp/simulations/abm_braunschweig.cpp

```diff
@@ -1022,7 +1022,7 @@ mio::IOResult<void> run(const std::string& input_file, const fs::path& result_di
         // Option to save the current run result to file
         if (save_result_result && save_single_runs) {
             auto result_dir_run = result_dir / ("abm_result_run_" + std::to_string(run_idx) + ".h5");
-            BOOST_OUTCOME_TRY(save_result(ensemble_results.back(), loc_ids, 1, result_dir_run.string()));
+            save_result_result = save_result(ensemble_results.back(), loc_ids, 1, result_dir_run.string());
```

Comment:
Why was this necessary? Looks like a nonintended (wrongly placed?) change here.

Reply:
The old version does produce a warning that save_result_result is uninitialized in the BOOST_OUTCOME_TRY a few lines below. Hence
> The change in abm_braunchschweig makes a lot of sense - I just removed the save_result_result in another branch as it was effectively unused, but stopping the entire ensemble because a singular output failed is likely not desirable. This here is better.